packages/generators/package.json

```diff
@@ -20,7 +20,8 @@
     "yeoman-generator": "4.7.2"
   },
   "peerDependencies": {
-    "webpack-cli": "3.x.x || 4.x.x"
+    "webpack": "4.x.x || 5.x.x",
+    "webpack-cli": "3.x.x || 4.x.x || 5.x.x"
```

Comment:
we don't have a v5 for cli

Reply:
@evenstensberg `webpack-cli@5` is listed as peerDependencies of `@webpack-cli/utils`: https://github.com/webpack/webpack-cli/blob/134c77a8e52ad17a73a8e61c0ce3ca8277ef2ddf/packages/utils/package.json#L25

I'll remove it from this PR. Should I remove it from the above as well?

Reply:
Yes

Reply:
Done